libraries/TGRSIint/TGRSIint.cxx

```diff
@@ -241,8 +245,8 @@ Int_t TGRSIint::TabCompletionHook(char* buf, int* pLoc, std::ostream& out){
 
 
 Long_t TGRSIint::ProcessLine(const char* line,Bool_t sync, Int_t *error) {
-  // If you print while fIsTabComplete is true, you will break tab complete.
-  // Any diagnostic print statements should be done after this if statement.
+  /// If you print while fIsTabComplete is true, you will break tab complete.
```

Comment:
I'm pretty sure these are actual comments for the code below, and not describing what the function does.